blocklist-client/src/common/mod.rs

```diff
@@ -1,21 +1,35 @@
-use serde::Serialize;
+use serde::{Deserialize, Serialize};
 use std::fmt;
-use warp::reject::Reject;
 
-#[derive(Debug)]
-pub enum Error {
-    AddressNotFound,
+pub mod error;
+
+/// The BlocklistStatus of a user address
+#[derive(Serialize, Deserialize)]
+pub struct BlocklistStatus {
+    // Whether the address is blocklisted or not
+    pub is_blocklisted: bool,
+    // The risk severity associated with an address
+    pub severity: String,
+    // Blocklist client's acceptance decision based on the risk severity of the address
+    pub accept: bool,
 }
 
-#[derive(Serialize)]
-pub struct ErrorResponse {
-    pub(crate) message: String,
+/// Risk severity linked to an address
+#[derive(Debug, PartialEq, Eq, Deserialize)]
+pub enum RiskSeverity {
```

Comment:
The implementations here are perfectly fine as is, but in the future you could also use something like [strum](https://github.com/Peternator7/strum) to automatically implement `Display` and `FromStr` for enums.